android_app/app/src/main/java/com/health/openscale/gui/measurement/ChartMeasurementView.java

```diff
@@ -76,6 +76,10 @@ public enum ViewMode {
     private boolean isInGraphKey;
     private ProgressBar progressBar;
 
+    private interface TrendlineComputationInterface {
+        public List<ScaleMeasurement> operation(List<ScaleMeasurement> measurementList);
```

Comment:
what does operation do in Java?

Reply:
An interface with a single method can be used to pass lambda into a method. https://www.w3schools.com/java/java_lambda.asp
It doesn't matter which name to use, so I renamed it to reduce confusion.

Reply:
Yeah indeed sorry I was a little bit confused I thought this were a new lambda function of Java 11 or something. :laughing: